notebooks/official/tabnet/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>Is it correct to say that both the training and the prediction data come from "Synthetic_2 (Syn2) data, described in Section 4.1 of the <a href="https://arxiv.org/pdf/1802.07814.pdf" target="_blank">Learning to Explain</a> paper"? gs://cloud-samples-data/ai-platform-unified/datasets/tabnet/tab_net_output/syn2 is the part of the data that is used for prediction?</p><p> </p><p>Could we also make the training data available on Cloud Storage?</p><p>What about the trained TabNet model, where is that stored?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>I uploaded the training data, too.</p><p>We don't make the model available. As our discussion with Helin Wang above, we try to get this PR check-in. I'll have a follow-up PR so that the user can get the output by themselves.</p>

Reply:
<p>I suggest the following wording:</p><p> </p><p>"Sample prediction data is stored on Google Cloud at gs://cloud-samples-data/ai-platform-unified/datasets/tabnet/tab_net_output/syn2. You can use your own set of prediction data, but you must ensure that the format of the prediction data is the same as the format of the training data."</p><p> </p><p>Information about the training set and the model are better suited for the Dataset section at the top of the notebook. </p>

Reply:
<p>Done. Thanks.</p>